PhysicsTools/TensorFlow/src/NTSession.cc

```diff
@@ -177,32 +171,9 @@ static NTSessionRegistrar registrar;
 
 std::atomic_int_fast64_t NTSession::step_id_counter_(1);
 
-// NOTE: On Android with a single device, there is never
-// a risk of an OpKernel blocking indefinitely:
-//
-// 1) No operations do I/O that depends on other simultaneous kernels,
-//
-// 2) Recv nodes always complete immediately: The inputs are sent into
-//    the local rendezvous before we start the executor, so the
-//    corresponding recvs will not block.
-//
-// Based on these assumptions, we can use the same thread pool for
-// both "non-blocking" and "blocking" OpKernels on Android.
-//
-// This may change down the road when we add support for multiple
-// devices that run concurrently, in which case we will need to
-// revisit this decision.
+// Override to allow CMSSW FWK to schedule
 void NTSession::SchedClosure(std::function<void()> c) {
-
-  // On Android, there is no implementation of ThreadPool that takes
```

Comment:
the notes about Android are in the upstream version.
Should they be put back?

Reply:
I'll add them back. Might be helpful when comparing revisions in future TF updates.